app/views/layouts/application.html.erb

```diff
@@ -17,6 +17,8 @@
     <%= stylesheet_link_tag 'application', media: 'all' %>
   </head>
 
+  <%= render partial: 'google_analytics' if google_analytics_enabled? %>
```

Comment:
As per discussion - seems googles docs are ambiguous but think this should be in head tag